dodola/services.py

```diff
@@ -154,6 +160,44 @@ def regrid(x, out, method, storage, weights_path=None, target_resolution=1.0):
     storage.write(out, regridded_ds)
 
 
+@log_service
+def clean_cmip6(x, out, storage, leapday_removal=True):
+    """Cleans and standardizes CMIP6 GCM
+
+    Parameters
+    ----------
+    x : str
+        Storage URL to input xr.Dataset that will be regridded.
+    out : str
+        Storage URL to write regridded output to.
```

Comment:
This refers to "regridding", should refer to "cleaning". This change is needed in both the `x` and `out` parameter docstr.